scripts/qa_crowbarsetup.sh

```diff
@@ -3640,6 +3640,11 @@ function get_horizon
                         ['elements']['$horizon-server']"`
     horizonserver=`resolve_element_to_hostname "$element"`
     horizonservice=`resolve_element_to_hostname "$element" service`
+    horizonprotocol=http
+    `crowbar $horizon proposal show default | \
+        rubyjsonparse "
+                    puts j['attributes']['$horizon']['apache']['ssl']"` && \
```

Comment:
Is the value that is being `puts` here expected to be a bash command?
If not, then this would not work.
```
`echo https` && echo doesntwork
```